dev-packages/node-integration-tests/suites/tracing/connect/scenario.js

```diff
@@ -13,7 +13,7 @@ Sentry.init({
 const connect = require('connect');
 const http = require('http');
 
-const init = async () => {
+const run = async () => {
```

Comment:
renaming these in a few places for consistency sake, init was a bit confusing IMHO.